web/ui/react-app/README.md

```diff
@@ -13,26 +13,26 @@ Instead of plain JavaScript, we use [TypeScript](https://www.typescriptlang.org/
 To work with the React UI code, you will need to have the following tools installed:
 
 * The [Node.js](https://nodejs.org/) JavaScript runtime.
-* The [Yarn](https://yarnpkg.com/) package manager.
+* The [Npm](https://www.npmjs.com/) package manager. Once you installed Node, npm should already be available.
 * *Recommended:* An editor with TypeScript, React, and [ESLint](https://eslint.org/) linting support. See e.g. [Create React App's editor setup instructions](https://create-react-app.dev/docs/setting-up-your-editor/). If you are not sure which editor to use, we recommend using [Visual Studio Code](https://code.visualstudio.com/docs/languages/typescript). Make sure that [the editor uses the project's TypeScript version rather than its own](https://code.visualstudio.com/docs/typescript/typescript-compiling#_using-the-workspace-version-of-typescript).
 
 **NOTE**: When using Visual Studio Code, be sure to open the `web/ui/react-app` directory in the editor instead of the root of the repository. This way, the right ESLint and TypeScript configuration will be picked up from the React workspace.
 
 ## Installing npm dependencies
 
-The React UI depends on a large number of [npm](https://www.npmjs.com/) packages. These are not checked in, so you will need to download and install them locally via the Yarn package manager:
+The React UI depends on a large number of [npm](https://www.npmjs.com/) packages. These are not checked in, so you will need to download and install them locally via the NPM package manager:
```

Comment:
They just call themselves "npm" officially, so I would just write it like that everywhere.